sdk/python/packages/flet-core/src/flet_core/cupertino_sliding_segmented_button.py

```diff
@@ -107,14 +107,20 @@ def _before_build_command(self):
         super()._before_build_command()
         self._set_attr_json("padding", self.__padding)
 
+    def did_mount(self):
+        super().did_mount()
+        assert (
+            len(self.__controls) >= 2
+        ), "CupertinoSlidingSegmentedButton must have at minimum two visible controls"
+
```

Comment:
Why do you do checks in `did_mount()`? Would it be better do that in `control.before_update()` (new method), *before* updating UI? `did_mount` is when control has already been added.